onnx/backend/test/case/node/convinteger.py

```diff
@@ -0,0 +1,116 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+
+class ConvInteger(Base):
+
+    @staticmethod
+    def export():  # type: () -> None
+
+        x = np.array([[[[0, 1, 2, 3, 4],  # (1, 1, 5, 5) input tensor
```

Comment:
Can use some other case (not symmetric tensor), at least to tell people this is a co-relation, not a real convolution?

Reply:
good point. changed  two cases.